flood_risk_engine.gemspec

```diff
@@ -1,69 +1,73 @@
-$LOAD_PATH.push File.expand_path("../lib", __FILE__)
+$LOAD_PATH.push File.expand_path("lib", __dir__)
 
 # Maintain your gem's version:
 require "flood_risk_engine/version"
 
 # Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name        = "flood_risk_engine"
-  s.version     = FloodRiskEngine::VERSION
-  s.authors     = ["Digital Services Team, EnvironmentAgency"]
-  s.email       = ["dst@environment-agency.gov.uk"]
-  s.homepage    = "https://github.com/defra/flood-risk-engine"
-  s.summary     = "FloodRiskEngine package containing core elements and functionality"
-  s.description = "FloodRiskEngine package containing core elements and functionality"
-  s.license     = "The Open Government Licence (OGL) Version 3"
+  s.name                  = "flood_risk_engine"
+  s.version               = FloodRiskEngine::VERSION
+  s.authors               = ["Digital Services Team, EnvironmentAgency"]
+  s.email                 = ["dst@environment-agency.gov.uk"]
+  s.homepage              = "https://github.com/defra/flood-risk-engine"
+  s.summary               = "FloodRiskEngine package containing core elements and functionality"
+  s.description           = "FloodRiskEngine package containing core elements and functionality"
+  s.license               = "The Open Government Licence (OGL) Version 3"
+  s.required_ruby_version = '>= 3.1'
 
   s.files = Dir["{app,config,db,lib}/**/*", "LICENSE", "Rakefile", "README.rdoc"]
-  s.test_files = Dir["spec/**/*"]
```

Comment:
Why has this been commented out?

Reply:
this seems to be a deprecated config: https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdeprecatedattributeassignment